### PR TITLE
Cals 5109 cws facilities

### PR DESCRIPTION
**Pull request by AlexanderSerbin**

### CODE REVIEW

File: src/main/java/gov/ca/cwds/cals/service/ChangedFacilityService.java [hunk 1]
```diff
@@ -44,12 +44,22 @@ public ChangedFacilityService() {
...
+  protected RecordChanges handleCwsCmsFacilityIds(Date dateAfter) {
     RecordChanges recordChanges = new RecordChanges();
-    recordChangeCwsCmsDao.streamChangedFacilityRecords(after).forEach(recordChanges::add);
+    recordChangeCwsCmsDao.streamChangedFacilityRecords(buildDateForCwsCms(dateAfter)).forEach(recordChanges::add);
     return recordChanges;
   }
 
+  private Date buildDateForCwsCms(Date date) {
+    Date dateAfter;
+    if (date == null) {
```
Comment (justajavaboy) [inline src/main/java/gov/ca/cwds/cals/service/ChangedFacilityService.java]: consider using ternary operator here

File: src/test/java/gov/ca/cwds/cals/web/rest/rfa/changed/ChangedFacilityResource.java [hunk 2]
```diff
@@ -44,13 +33,27 @@
...
+  ) throws ParseException {
     List<ChangedFacilityDTO> changedFacilityDTOList = changedFacilityService
-        .changedFacilitiesStream(
-            dateAfter, lisDateAfter).collect(Collectors.toList());
+            .changedFacilitiesStream(
+                    null, getDateFromString(stringLisDateAfter)).collect(Collectors.toList());
     return new CollectionDTO<>(changedFacilityDTOList);
   }
+
+  private Date getDateFromString(@QueryParam(LIS_DATE_AFTER) String stringLisDateAfter) throws ParseException {
```
Comment (justajavaboy) [inline src/test/java/gov/ca/cwds/cals/web/rest/rfa/changed/ChangedFacilityResource.java]: consider moving this method to Utils class

File: src/main/java/gov/ca/cwds/cals/service/ChangedFacilityService.java (hunk 1 above)
Comment (justajavaboy) [reply src/main/java/gov/ca/cwds/cals/service/ChangedFacilityService.java]: all code in method can be refactored in one line

Comment (AlexanderSerbin) [reply src/main/java/gov/ca/cwds/cals/service/ChangedFacilityService.java]: Agree. Good point

File: src/test/java/gov/ca/cwds/cals/web/rest/rfa/changed/ChangedFacilityResource.java (hunk 2 above)
Comment (AlexanderSerbin) [reply src/test/java/gov/ca/cwds/cals/web/rest/rfa/changed/ChangedFacilityResource.java]: Agree

**Pull request merged**